src/main/java/net/zhuoweizhang/raspberryjuice/RemoteSession.java

```diff
@@ -222,43 +283,123 @@ protected void handleCommand(String c, String[] args) {
 				
 			// events.clear
 			} else if (c.equals("events.clear")) {
-				interactEventQueue.clear();
-				chatPostedQueue.clear();
+				int entityId = -1;
+				if (args[0].length() > 0)
+					entityId = Integer.parseInt(args[0]);
+				if (entityId == -1) {
+					interactEventQueue.clear();
+					chatPostedQueue.clear();
+					projectileHitQueue.clear();
+				}
+				else {
```

Comment:
What is the use case for removing events for a specific entity?

If this is required, I would suggest a new command, maybe `entity.events.clear` (?)

Reply:
Because when you are not alone, and you have multiple childrens running their code at the same time, you do not want to clear all events of all players. In sigleplayer scenario it's fine, but my goal was to also allow clear events for specified player only.

Reply:
About entity.events.clear instead of modify events.clear - seems fine. I will change it.